packages/x-tree-view/src/internals/plugins/useTreeViewIcons/useTreeViewIcons.test.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import {
+  describeTreeView,
+  DescribeTreeViewRendererReturnValue,
+} from 'test/utils/tree-view/describeTreeView';
+import {
+  UseTreeViewExpansionSignature,
+  UseTreeViewIconsSignature,
+} from '@mui/x-tree-view/internals';
+
+describeTreeView<[UseTreeViewIconsSignature, UseTreeViewExpansionSignature]>(
+  'useTreeViewIcons plugin',
+  ({ render }) => {
+    describe('slots (expandIcon, collapseIcon, endIcon, icon)', () => {
+      const getIconTestId = (response: DescribeTreeViewRendererReturnValue<[]>, itemId: string) =>
+        response.getItemIconContainer(itemId).querySelector(`div`)?.dataset.testid;
+
+      it('should render the expandIcon slot defined on the tree if no icon is defined on the item and the item is collapsed', () => {
```

Comment:
```suggestion
      it('should render the expandIcon slot defined on the tree if no icon slot is defined on the item and the item is collapsed', () => {
```
Could be more explicit like thi?